sandbox/color_names.py

```diff
@@ -35,8 +34,7 @@ def on_mount(self):
     def compose(self) -> ComposeResult:
         for color_name, color in COLOR_NAME_TO_RGB.items():
             color_placeholder = ColorDisplay(name=color_name)
-            is_dark_color = sum(color) < 400
-            color_placeholder.styles.color = "white" if is_dark_color else "black"
+            color_placeholder.styles.color = "auto 90%"
```

Comment:
This "is_dark_color" test is no longer required, since the new `auto` property does exactly this :blush: 

The result:
![Screenshot from 2022-06-29 14-33-56](https://user-images.githubusercontent.com/722388/176456021-8a3c7fd5-a586-4bac-bbef-ab38bea49054.png)